Author: Adam Gandelman <adamg@canonical.com>
Subject: Temporarily fix doc building

Upstream commit b057ab60a39eb8dfa6604d6f207625c29df6cd70 broke doc builds.
Fix locally until fix is committed upstream

Index: nova-2012.1/doc/source/conf.py
===================================================================
--- nova-2012.1.orig/doc/source/conf.py	2012-03-12 13:23:19.000000000 -0700
+++ nova-2012.1/doc/source/conf.py	2012-03-12 13:42:46.394307108 -0700
@@ -114,7 +114,7 @@
 # List of tuples 'sourcefile', 'target', u'title', u'Authors name', 'manual'
 
 man_pages = [
-    ('man/novamanage', 'novamanage', u'Cloud controller fabric',
+    ('man/nova-manage', 'nova-manage', u'Cloud controller fabric',
      [u'OpenStack'], 1)
  ]
 
